content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,208 @@
+from being stored in close proximity by directing them to different locations determined by the
+warehouse manager.
 
-Configuration
-=============
+.. seealso::
+   - :ref:`Routes-- how products move<inventory/routes/use_routes/push>`
+   - :ref:`Warehouse locations<inventory/warehouse_location/definition>`
 
-In the *Inventory* app, go to :menuselection:`Configuration --> Settings` and activate the
-*Multi-Step Routes*. By doing so, the *Storage Locations* will be automatically activated.
+Enable putaway rules
+====================
 
-.. image:: putaway/putaw1.png
+Begin by navigating to :menuselection:`Inventory --> Configuration --> Settings`, then activate the
+:guilabel:`Multi-Step Routes` checkbox under the :guilabel:`Warehouse` heading. By doing so, the
+:guilabel:`Storage Locations` setting is automatically enabled as well. Finally, click
+:guilabel:`Save`.
+
+.. image:: putaway/multi-step-routes.png
    :align: center
+   :alt: Check multi-step routes box
+
+Put away products
+=================
 
-Setting up a Putaway Rule
-=========================
+Putaway rules are commonly used to move products to specific locations for easy retrieval. Thus,
+products are grouped in the same :guilabel:`Product Category` when a single putaway rule captures
+the group's storage needs.
 
-In some cases, like for a retail shop storing vegetables and fruits, we have to store products in
-different locations to maintain product quality.
+To manage where specific products are routed for storage, navigate to :menuselection:`Inventory -->
+Configuration --> Putaway Rules`. Use the :guilabel:`Create` button to configure a new putaway rule
+on a :guilabel:`Product` and/or :guilabel:`Product Category` that the rule affects.
 
-Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations
-*WH/Stock/Vegetables* and *WH/Stock/Fruits*.
+In the same line, the :guilabel:`When product arrives in` location is where the putaway rule is
+triggered to create an operation to move the product to the :guilabel:`Store to` location. The
+:guilabel:`Store to` location must be sublocation of the first (ie. `WH/Stock/Fruits` is a specific,
+named location inside `WH/Stock` to make the products stored here easier to find).
 
-To manage those locations, we will create putaway rules. To do so, open the *Inventory* app and go
-to :menuselection:`Configuration --> Putaway Rules`. Then, click on create and configure your first
-rule indicating the main location the product will enter before being redirected to the right
-location.
+Ensure all apples are stored in the fruits section by filling field :guilabel:`Store to` with the
+location `WH/Stock/Fruits` when the :guilabel:`Product`, `Apple` arrives in `WH/Stock`. Repeat this
+for all products and hit :guilabel:`Save`.
+
+.. image:: putaway/create-putaway-rules.png
+   :align: center
+   :alt: Create putaway rules for apples and carrots
```

Comment:
I think this should be in an Example box. Also, "alt" image descriptions should end in a period.
```suggestion
.. example::
   Ensure all apples are stored in the fruits section by filling field :guilabel:`Store to` with the location `WH/Stock/Fruits` when the :guilabel:`Product`, `Apple` arrives in `WH/Stock`. Repeat this for all products and hit :guilabel:`Save`.

   .. image:: putaway/create-putaway-rules.png
      :align: center
      :alt: Create putaway rules for apples and carrots.
```